xtask/src/main.rs

```diff
@@ -34,6 +34,8 @@ impl Workspace {
 }
 
 fn run_action(workspace: Workspace, target: &str, extra_args: &[String], features: &[String]) {
+    println!("Running {} in {:?} workspace...", target, workspace);
```

Comment:
When does this print stuff out and does it add something useful? I must admit to a general allergy to more stuff being added to the terminal.

Reply:
I knew you'd say this! Which is why it's in a separate commit (1fef78f) with justification!

Reply:
Yes, but it doesn't answer the real question :) Is it useful, or is it "just another" not very relevant piece of information? In other words, is it ever useful for me to know *which* workspace `build`/`test` (etc.) is being run in?

Reply:
I find it useful, but that argument hasn't worked for me in the past! I've backed it out.